src/_scss/pages/interactiveDataSources/_scroller.scss

```diff
@@ -0,0 +1,39 @@
+$top: 60px;
+
+.scroller-container {
+  display: block;
+  position: relative;
+  width: 100%;
+}
+
+.sk-scroller-content-collection {
+  display: none;
+}
+
+.scroller-backdrop  {
+    position: sticky;
+    display: block;
+    top: $top;
+    overflow: hidden;
+  }
+
+.scroller-backdrop > * {
+  width: 100%;
+  height: calc(100vh - #{$top});
+  margin: 0 auto;
+}
+
+.scroller-backdrop div {
+    border: none;
+  }
```

Comment:
Weird spacing here - is there some sort of autoformatting or linting you can run to catch things like this?

Reply:
good catch, we have eslint in place but nothing for scss